.github/workflows/build.yml

```diff
@@ -65,34 +65,85 @@ jobs:
       run: echo 'cmake --build ${{github.workspace}}/build --target bootcd --target livecd' | ${{github.workspace}}/RosBE-CI/RosBE.sh . 0 ${{matrix.arch}}
     - name: Print ccache statistics
       run: ccache -s
-    - name: Upload bootcd
```

Comment:
While here, from previous commit,
`if: steps.get_rosbe.outputs.cache-hit != 'true'`
and
`if: matrix.compiler == 'gcc'`
miss
`${{ ... }}`